docs/docs-content/enterprise-version/install-palette/airgap/airgap.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_label: "Airgap"
+title: "Airgap"
+description: "Learn about Palette in an airgap environment and how to install Palette in an airgap environment."
+icon: ""
+sidebar_position: 10
+hide_table_of_contents: false
+tags: ["palette", "self-hosted", "airgap"]
+---
+
+
+You can install Palette in an airgapped environment. An airgap environment lacks direct access to the internet and is intended for environments with strict security requirements.
```

Comment:
I noticed that sometimes doc refers to `air-gapped` and here it's `airgapped`

Reply:
Hmm...it should be airgapped. I'll snag this in a separate PR. Thank you @cloudmaniac for calling it out.